routes/spec.js

```diff
@@ -847,6 +847,162 @@ Consider supporting The Slothpixel Project on Patreon to help cover the hosting
+                      items: {
+                        type: 'string',
+                      },
+                    },
+                    ranks: {
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        properties: {
+                          name: {
+                            type: 'string',
+                          },
+                          permissions: {
+                            type: 'array',
+                            items: {
+                              type: 'number',
+                            },
+                          },
+                          default: {
+                            type: 'boolean',
+                          },
+                          tag: {
+                            type: 'string',
+                          },
+                          created: {
+                            type: 'integer',
+                          },
+                          priority: {
+                            type: 'integer',
+                          },
+                        },
+                      },
+                    },
+                    members: {
+                      description: 'Array playerof players on the guild',
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        properties: {
+                          uuid: {
+                            description: 'Player UUID',
+                            type: 'string',
+                          },
+                          rank: {
+                            description: 'Player rank in the guild',
+                            type: 'string',
+                          },
+                          joined: {
+                            description: 'Member join date',
+                            type: 'integer',
+                          },
+                          quest_participation: {
+                            description: 'How many much the member has contributed to guild quests',
+                            type: 'integer',
+                          },
+                          exp_history: {
+                            description: 'Contains raw guild xp earned in the past week. Uses format YYYY-MM-DD.',
+                            type: 'object',
+                          },
+                          muted_till: {
+                            description: 'Date the member is muted until',
+                            type: 'integer',
+                          },
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+        route: () => '/guilds/name/:name',
+        func: async (request, response, callback) => {
+          try {
+            const guild = await getGuildFromName(request.params.player);
```

Comment:
You need to put `request.params.name` here, and also use the correct parameter in the graphql processor

Reply:
sounds good!

Reply:
You should also add a graphql spec like here https://github.com/slothpixel/core/blob/master/routes/spec.graphql#L29

Reply:
Alright!